cypress/e2e/catalogue/catalogueItems.cy.ts

```diff
@@ -513,4 +513,67 @@ describe('Catalogue Items', () => {
 
     cy.url().should('contain', 'catalogue/items/6');
   });
+
+  it('can move multiple catalogue items', () => {
+    cy.visit('/catalogue/5');
+
+    cy.findAllByLabelText('Toggle select row').first().click();
+    cy.findAllByLabelText('Toggle select row').eq(2).click();
+
+    cy.findByRole('button', { name: 'Move to' }).click();
+
+    cy.findByRole('button', { name: 'Move here' }).should('be.disabled');
+
+    cy.findByText('Energy Meters V2').click();
+
+    cy.startSnoopingBrowserMockedRequest();
+
+    cy.findByRole('button', { name: 'Move here' }).click();
+
+    cy.findBrowserMockedRequests({
+      method: 'PATCH',
+      url: '/v1/catalogue-items/:id',
+    }).should(async (patchRequests) => {
+      expect(patchRequests.length).equal(2);
+      expect(JSON.stringify(await patchRequests[0].json())).equal(
+        JSON.stringify({ catalogue_category_id: '8967' })
+      );
+      expect(JSON.stringify(await patchRequests[1].json())).equal(
+        JSON.stringify({ catalogue_category_id: '8967' })
+      );
+    });
+  });
+
+  it('errors when moving multiple catalogue items to a catalogue category with different catalogue item properties ', () => {
+    cy.visit('/catalogue/5');
+
+    cy.findAllByLabelText('Toggle select row').first().click();
+    cy.findAllByLabelText('Toggle select row').eq(2).click();
+
+    cy.findByRole('button', { name: 'Move to' }).click();
+
+    cy.findByText('Cameras').click();
+
+    cy.findByRole('button', { name: 'Move here' }).click();
+
+    cy.findByRole('dialog')
+      .should('be.visible')
+      .within(() => {
+        cy.contains(
+          'The destination catalogue item properties must precisely match the current destination. Ensure identical attributes, order, and formatting, with no spacing variations.'
+        );
+      });
+  });
+  it('Move here button is disabled when trying to move to the current location', () => {
```

Comment:
I think there should be either an e2e test or unit test to ensure the Move button is also disabled when attempting to move into a catalogue category that isn't a leaf.

Reply:
That is already tested. When I checked if the button is disabled when the dialog is opened because it opens on a non leaf node

Reply:
Doesn't that just mean its testing that its disabled if moving to the same place, rather than if the category is a leaf?

Reply:
No, because when a user opens the dialog is opens to the parent catalogue category and not the catalogue category that it is currently in